packaging/appimage/org.gaphor.Gaphor.appdata.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Isn't this the same file we use for Flatpak?

Reply:
Yes it is, any ideas on how we could keep the files common across the repos? The only way I know is to put these files in a git submodule.

Reply:
Maybe we could a checkout of the Gaphor repo from the build script in our flatpak repo. I suppose multi-repo checkouts are supported, as they are for Azure Pipelines.

Reply:
I like this idea! Flathub uses a buildbot and I'm not sure how configurable things are from our repo. I will look in to this more.

Reply:
I looked into whether we can have the build files (appdata.xml and desktop) centralized a bit more. It looks like projects like [Lollypop](https://gitlab.gnome.org/World/lollypop) are using meson to build the project and move the files needed since flatpak-builder supports meson. Then all the files we have in the share folder of the flathub repo could be moved to our main repo. I think this would be a whole new project to move things to meson, so I suggest we a separate PR for that.